addons/bus/static/src/js/services/assets_watchdog_service.js

```diff
@@ -8,7 +8,7 @@ export const assetsWatchdogService = {
 
     start(env, { notification }) {
         const assets = {};
-        let assetsChangedNotificationId = null;
+        let assetsChangedNotificationRemove = null;
```

Comment:
no need to store the callback. it should instead be just a boolean. something like
```js
let isNotificationDisplayed = false;
```